lib/dataTrigger/DataTriggerExecutionResult.js

```diff
@@ -0,0 +1,32 @@
+class DataTriggerExecutionResult {
+  /**
+   * @param {DataTriggerExecutionError[]} errors
+   */
+  constructor(errors = []) {
+    this.errors = errors;
+  }
+
+  /**
+   * @param {DataTriggerExecutionError} error
+   */
+  addError(error) {
```

Comment:
Don't we want to have the same interface as for `ValidationResult` https://github.com/dashevo/js-dpp/blob/0bf88798feee663be6bc61859f273a13b7ee8d7c/lib/validation/ValidationResult.js#L14

Reply:
Yeah, sure